client/js/Deck.js

```diff
@@ -4,12 +4,13 @@
  */
 
 class Card {
-  constructor (cardSuit, cardValue) {
+  constructor (cardSuit, cardValue, cardId) {
     this.suit = cardSuit
     this.value = cardValue
+    this.id = cardId
   }
 }
-
+i = 1  //Globally declared i for card Ids
```

Comment:
*Never* use ambiguous variable names globally. Infact, here, since we have numerous files, keeping track of any global variable will be a pain, hence avoid using them as far as possible. Instead, pass them as arguments to whichever functions needed.